onnxruntime/test/providers/cpu/math/topk_op_test.cc

```diff
@@ -456,5 +456,35 @@ TEST(TopKOperator, SortedSelection) {
   RunTest(11, 5, input_vals, input_dimensions, expected_vals, expected_indices, expected_dimensions, false, axis, 0);  // smallest values
 }
 
+TEST(TopKOperator, BigArrayTopKSorted) 
+{
+  std::vector<float> input_vals(10000, 0.0f);
+  std::iota(input_vals.begin(), input_vals.end(), 0.0f);
+  std::vector<int64_t> input_dimensions = {10000};
+  std::vector<float> expected_vals(1000, 0.0f);
+  std::iota(expected_vals.begin(), expected_vals.end(), 9000.0f);
+  std::reverse(expected_vals.begin(), expected_vals.end());
+  std::vector<int64_t> expected_indices(1000, 0);
+  std::iota(expected_indices.begin(), expected_indices.end(), 9000);
+  std::reverse(expected_indices.begin(), expected_indices.end());
+  std::vector<int64_t> expected_dimensions = {1000};
+  RunTest(11, 1000, input_vals, input_dimensions, expected_vals, expected_indices, expected_dimensions, false, 0, 1, 1);
+}
+
+TEST(TopKOperator, BitArrayBigTopKSorted) 
```

Comment:
>BitArrayBigTopKSorted [](start = 19, length = 21)

BigArrayBigTopKSorted?

Reply:
Sorry just a typo, renaming to BigArray...